src/datanode/src/instance.rs

```diff
@@ -330,6 +334,10 @@ impl Instance {
         self.procedure_manager
             .start()
             .context(StartProcedureManagerSnafu)?;
+        let _ = self
```

Comment:
So could we add an info log to record the error?

Reply:
We don't want to bother users, so we'll consider not printing any logs related to version reporting.

Reply:
@paomian `Debug` log is acceptable.